saleor/graphql/product/tests/test_product.py

```diff
@@ -3480,6 +3480,48 @@ def test_products_query_with_filter_stock_availability_as_staff(
     assert len(products) == 3
 
 
+def test_products_query_with_filter_stock_availability_including_reservations(
+    query_products_with_filter,
+    staff_api_client,
+    product_list,
+    order_line,
+    checkout_line,
+    permission_manage_products,
+    channel_USD,
+):
+
```

Comment:
Please use `given`, `when`, `then`. As it's mentioned in the [contribution guide.](https://docs.saleor.io/docs/3.x/developer/community/contributing#given-when-then)